src/components/molecule/PieChartView.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { VictoryPie, VictoryContainer } from 'victory'
+
+interface Idata {
+    angle?: number,
+    radius?: number,
+    label?: string,
+    subLabel?: string,
+    fontSize?: number,
+    style?: object
+    quarter?: string,
+    earnings?: number,
+    x?: number,
+    y?: number
+}
+
+export const PieChartView = () => {
+    const data: Idata[] = [
+        { quarter: 'Car', earnings: 13000 },
+        { quarter: 'Pedestrian', earnings: 16500 },
+        { quarter: 'Cyclist', earnings: 14250 },
+        { quarter: 'Public Transport', earnings: 19000 },
+        { quarter: 'Train', earnings: 19000 },
+        { quarter: 'Light Train', earnings: 19000 }
+    ]
+    return (
+        <React.Fragment>
+            <VictoryPie data={data} x='quarter' y='earnings'
```

Comment:
Why are these props in the same line?

Reply:
Personally i think it's OK for props to be in a single  as long as there are only few of them.
What's consider "a few"? well, that is  a good question. 

In any case, until we decide on a style-guide for this project, I think we shouldn't prevent merging because of style issues (We should comment on them tough)